manimo/actuators/arms/franka_arm.py

```diff
@@ -256,7 +257,10 @@ def step(self, action):
 
         if self.action_space == ActionSpace.Cartesian:
             if self.ik_mode == IKMode.Polymetis:
-                self._apply_eef_commands(action)
+                self.robot.move_to_ee_pose(action[:3], action[3:], delta=False, time_to_go=None, op_space_interp=False)
```

Comment:
can you make this a function, either change the original function or make a new one.

Reply:
updated!